src/Aspose.Imaging.Cloud.Sdk/Model/Requests/UpdateImageRequest.cs

```diff
@@ -43,27 +43,27 @@ public UpdateImageRequest()
         /// Initializes a new instance of the <see cref="UpdateImageRequest"/> class.
         /// </summary>
         /// <param name="name">Filename of an image.</param>
-        /// <param name="format">Resulting image format. Please, refer to https://docs.aspose.cloud/display/imagingcloud/Supported+File+Formats#SupportedFileFormats-CommonOperationsFormatSupportMap for possible use-cases.</param>
         /// <param name="newWidth">New width of the scaled image.</param>
         /// <param name="newHeight">New height of the scaled image.</param>
         /// <param name="x">X position of start point for cropping rectangle.</param>
         /// <param name="y">Y position of start point for cropping rectangle.</param>
         /// <param name="rectWidth">Width of cropping rectangle.</param>
         /// <param name="rectHeight">Height of cropping rectangle.</param>
         /// <param name="rotateFlipMethod">RotateFlip method (Rotate180FlipNone, Rotate180FlipX, Rotate180FlipXY, Rotate180FlipY, Rotate270FlipNone, Rotate270FlipX, Rotate270FlipXY, Rotate270FlipY, Rotate90FlipNone, Rotate90FlipX, Rotate90FlipXY, Rotate90FlipY, RotateNoneFlipNone, RotateNoneFlipX, RotateNoneFlipXY, RotateNoneFlipY). Default is RotateNoneFlipNone.</param>
+        /// <param name="format">Resulting image format. Please, refer to https://docs.aspose.cloud/display/imagingcloud/Supported+File+Formats#SupportedFileFormats-CommonOperationsFormatSupportMap for possible use-cases.</param>
         /// <param name="folder">Folder with image to process.</param>
         /// <param name="storage">Your Aspose Cloud Storage name.</param>
-        public UpdateImageRequest(string name, string format, int? newWidth, int? newHeight, int? x, int? y, int? rectWidth, int? rectHeight, string rotateFlipMethod, string folder = null, string storage = null)             
+        public UpdateImageRequest(string name, int? newWidth, int? newHeight, int? x, int? y, int? rectWidth, int? rectHeight, string rotateFlipMethod, string format = null, string folder = null, string storage = null)             
```

Comment:
Is it really necessary to change the order of parameters affecting API use cases ?

Reply:
Yes, because parameter with default value must be after parameters without default values in dotnet

Reply:
According to C# specification, all optional parameters must be defined at the end of the parameter list

Reply:
In such case the [examples](https://github.com/aspose-imaging-cloud/aspose-imaging-cloud-dotnet/tree/master/Examples) should be updated as well, isn't it ?
Also [documentation](https://docs.aspose.cloud/display/imagingcloud/Developer+Guide)  should be reviewed to aware these changes.

Reply:
Yes, we will update documentation and examples